src/inmanta/agent/agent.py

```diff
@@ -524,6 +476,9 @@ def __init__(self, process: "Agent", name: str, uri: str, *, ensure_deploy_on_st
 
         # init
         self._nq = ResourceScheduler(self, self._env_id, name)
+        self.work_queue = scheduler.TaskQueue()
+        self.work_queue_drainer = scheduler.TaskRunner(self.work_queue)
```

Comment:
Why have two different terms for the same thing (`queue_drainer` vs. `TaskRunner`)?

Reply:
changed